agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -255,9 +255,13 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
               if ("none".equals(otelConfig.getString("otel.traces.exporter"))) {
                 // in this case the spanExporter here is the noop spanExporter
                 return spanExporter;
-              } else {
+              } else if (!"otlp".equals(otelConfig.getString("otel.traces.exporter"))) {
+                startupLogger.verbose(
+                    "Unsupported otel.traces.exporter: {}",
+                    otelConfig.getString("otel.traces.exporter"));
                 return wrapSpanExporter(spanExporter, configuration);
```

Comment:
I forget, why do we not want to wrap the `otlp` exporter?

Reply:
"wrap" was for telemetry processor, which is not available for otlp.

Reply:
> which is not available for otlp

why not?

Reply:
java agent's telemetry processor, we want to apply to otel metrics to otlp endpoint?

Reply:
let's chat after this azmon meeting